src/App/src/Common/Pagination.php

```diff
@@ -18,18 +16,18 @@ class Pagination
      */
```

Comment:
```suggestion
     */
```
Please remove the entire comment.